pnpm-lock.yaml

```diff
@@ -279,7 +279,7 @@ importers:
         version: link:../app
       '@shopify/cli-hydrogen':
         specifier: 9.0.2
-        version: 9.0.2(react-dom@18.2.0)(react@18.2.0)
```

Comment:
I don't think this file should have changed?

Reply:
I honestly have no idea sometimes why these things change... maybe my local version of pnpm is slightly off? But I'm on 8.15.7 so 🤷‍♂️